econml/dml/dynamic_dml.py

```diff
@@ -0,0 +1,789 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Given that we plan on adding other approaches too for dynamic panel learning (e.g. a doubly robust one for discrete treatments and other time-varying methods), it might be better to place `dynamic_dml` in a new module: `dynamic`, e.g.
`from econml.dynamic import DynamicDML`
or
`from econml.dynamic.dml import DynamicDML`

Reply:
Done